content/en/blogs/TiDB-on-KubeSphere-using-qke.md

```diff
@@ -218,10 +220,12 @@ I hope you guys all have successfully deploy TiDB. Both TiDB and KubeSphere are
 
 ## References
 
-https://docs.pingcap.com/tidb-in-kubernetes/stable/get-started
+**KubeSphere GitHub**: https://github.com/kubesphere/kubesphere 
+
+**TiDB GitHub**: https://docs.pingcap.com/tidb-in-kubernetes/stable/get-started
```

Comment:
This link is not TiDB GitHub. It is actually TiDB Documentation

Reply:
Updated